{{ cookiecutter.__package_name_kebab_case }}/Dockerfile

```diff
@@ -53,13 +53,14 @@ RUN --mount=type=cache,uid=$UID,gid=$GID,target=/home/app/.cache/pypoetry/ \
 
 FROM base as ci
 
-# Install git so we can run pre-commit.
+# Allow CI to run as root.
 USER root
+
+# Install git so we can run pre-commit.
```

Comment:
FYI: Not running as `root` in GitLab CI/CD seems to be quite difficult. For now, we can run as `root`, but in the future we should align GitLab CI/CD with the GitHub Actions workflow where we don't enter the dev container but rather call it with the Dev Containers CLI.

Reply:
Makes sense!